oas

```diff
@@ -142,6 +142,65 @@ paths:
               schema:
                 $ref: '#/components/schemas/Animal'
 
+  /events:
+    get:
+      summary: Get events for the animals
+      tags:
+        - Animal Information
+      parameters:
+        - name: date_start
+          in: query
+          required: true
+          schema: 
+            type: string
+            format: date-time
+        - name: date_end
+          in: query
+          description: default is NOW
+          schema: 
+            type: string
+            format: date-time
+        - name: animal_urn
+          in: query
+          schema: 
+            type: string
+        - name: category
+          in: query
+          schema: 
+            $ref: '#/components/schemas/EventCategory'
+      responses:
+        200:
+          description: 'successful operation'
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Event'
+  /weights:
```

Comment:
I don't see the possibility of adding weight info (no POST). Is that deliberate?
If I had an external balance, independent of my FMIS, I would not be able to "plug in" my information

Reply:
Your response about "our own implementation" is a bit problematic. The service template is meant to be implementation-agnostic. Having a POST API would provide a generic method for any implementation to integrate with "smart scales" (or manual inputs); not having it makes scale integration proprietary to each implementation "by design"

Reply:
Of course, we build our own implementation completely following this service template. Why is this problematic? I think the overall goal is that as much parties as possible implementing this service-templates. So in that case I see a perfect use-case for an ATLAS cooperation: The scaler implements this service, we build a client and GET the data from them. We also implement this service and provide aggregated data from different sources for other parties. No proprietary communication at all.
In that case nobody needs to inject data into a POST endpoint. Everyone providing this kind of data should implement this service. That brings the users much better control over their data.